src/main/java/WorkShop01.kt

```diff
@@ -41,21 +52,29 @@ object WorkShop01 {
     // PS2. Оператор, который определяет остаток от деления - %
     // /
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P5Test
-    fun evenOddNumber(number: Int) {
-        TODO()
+    fun evenOddNumber(number: Int): String {
+        return if (number % 2 == 0) {
+            "Чётное"
+        } else "Нечётное"
     }
 
     //TODO 6. Необходимо реализовать лямбда-функцию и задать ей имя lambda, которая
     // 1. Принимает два значения типа Int и возвращает значение типа Int
     // 2. Результат выполнения лямбда-функции - сравнить два значения и вернуть максимальное из 2-х
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P6Test
-    //val lambda: TODO()
+    val lambda: (Int, Int) -> Int = {num1, num2 ->
```

Comment:
Чтобы применить автоформатирование в Intelij нужно выделить фрагмент кода и нажать ctrl+alt+L
и тогда все сразу станет на свои места:)
```suggestion
    val lambda: (Int, Int) -> Int = { num1, num2 ->
```

Reply:
моя жизнь не будет прежней :) спасибо!